kasa/smart/modules/lighttransition.py

```diff
@@ -181,3 +181,13 @@ def query(self) -> dict:
             return {}
         else:
             return {self.QUERY_GETTER_NAME: None}
+
+    async def _check_supported(self):
```

Comment:
We could probably generalize this at some point by checking if the query is `{}` and if a key defined in a class variable is available in the sysinfo data?

Not needed for this PR though, but just something that we may want to revisit later on.